File: platform/sdk/python/examples/graph/graph-001_cdc_new.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.cdcNew("consumer-001", {"offset": 0})
if not result["ok"] or result["consumerId"] != "consumer-001":
    sys.exit(1)
print("XJ-OK: graph-001_cdc_new")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-001_cdc_new.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.cdcNew("consumer-001", { offset: 0 });
if (!result.ok || result.consumerId !== "consumer-001") process.exit(1);
console.log("XJ-OK: graph-001_cdc_new");
process.exit(0);
